pkg/agent/flags.go

```diff
@@ -0,0 +1,13 @@
+package agent
+
+type flagInt interface {
+	DebugServerData() bool
+}
+
+var Flags flagInt = &initialFlagTest{}
```

Comment:
The goal with this is to make it so that select flags are globally available? If so, I like it, better than threading flags through all the time.

Reply:
Bigger discussion -- https://github.com/kolide/launcher/issues/1029